packages/project-disaster-trail/src/components/Game/TaskScreen/SolveScreen.js

```diff
@@ -69,49 +71,41 @@ class SolveScreen extends PureComponent {
   };
 
   render() {
-    const { completeActiveTask, activeTask, playerKitItems } = this.props;
+    const { completeActiveTask, open, activeTask } = this.props;
     const { correctItemsChosen } = this.state;
+    // eslint-disable-next-line react/destructuring-assignment
+    const activeTaskInState = this.state.activeTask;
```

Comment:
Is there a reason you want to avoid destructuring? `this.state` destructuring is happening in the line right above

Reply:
Good question! There's already an `activeTask` that we're getting from props, so I wasn't able to reuse the variable.

This is because `this.props.activeTask` is null as soon as the active task is complete. That would then re-render the component with no active task content, which would be jarring while the component animates off-screen.

So I'm stashing the `activeTask` in state as well.